aip/general/0157.md

```diff
@@ -56,60 +80,21 @@ enum BookView {
   conflict. In this situation, the service **should** nest the view enum within
   the individual resource.
 
-### Read masks
+### Read masks as a request field
 
-Read masks are useful for granting the user fine-grained control over what
-fields are returned, and are specified as a single field on the request
+Similar to a system parameter, an API **may** support read masks as a single field on the request
 message: `google.protobuf.FieldMask read_mask`.
 
-- The field mask **must** be a `google.protobuf.FieldMask` and **should** be
+- The read mask **must** be a `google.protobuf.FieldMask` and **should** be
   named `read_mask`.
-- The field mask **should** be optional, and the API **should** provide and
-  document an appropriate default if the field mask is not provided.
+- The field mask **should** be optional:
   - An explicit value of `"*"` **should** be supported, and **must** return all
     fields.
-  - The default value **may** be something other than `"*"` (for example, if
-    certain fields are expensive to compute and it is preferable to exclude
-    them by default).
-  - The field mask **may** be designated as required if necessary.
-  - The default for `List` and `Get` **may** be different from each other.
-    However, if they are, the default for `Get` **must** be a (non-strict)
-    superset of the default for `List`.
-  - While it is less common, read masks **may** be used on methods other than
-    `List` and `Get`.
+  - The default value **must** be`"*"` (if the field mask is not provided, all fields **must** be returned).
```

Comment:
Here we dictate the default value **must** be `"*"` but in the System parameter section above it says the default **should** be `"*"`. Should we be consistent between the two?